model/lifecycle.go

```diff
@@ -281,6 +282,8 @@ func RestartVersion(versionId string, taskIds []string, abortInProgress bool, ca
 		}
 	}
 
+	grip.Infof("Time to archive: %s", time.Now().Sub(t0).String())
```

Comment:
You can use time.Since to make it a bit more succinct. I would also log discreet fields with message.Fields (it'll make searching for this easier)